README.md

```diff
@@ -113,13 +113,30 @@ A retry strategy let you specify when request-retry should retry a request
  * @param  {Null | Object} err
  * @param  {Object} response
  * @param  {Object} body
+ * @param  {Object} options copy 
  * @return {Boolean} true if the request should be retried
  */
-function myRetryStrategy(err, response, body){
+function myRetryStrategy(err, response, body, options){
   // retry the request if we had an error or if the response was a 'Bad Gateway'
   return err || response.statusCode === 502;
 }
 
+/**
+ * @param  {Null | Object} err
+ * @param  {Object} response
+ * @param  {Object} body
+ * @param  {Object} options copy 
+ * @return {Object} mustRetry: {Boolean} true if the request should be retried
+ *                  options: {Object} new options for request
```

Comment:
`options: {Object} optional, if specified this options object will be used for the retried request`